Resources/config/yui.yml

```diff
@@ -297,6 +301,23 @@ system:
                 dashboardview-ez-template:
                     type: 'template'
                     path: %ez_platformui.public_dir%/templates/dashboard.hbt
+                ez-dashboardblocksview:
+                    requires:
+                        - 'ez-templatebasedview'
+                        - 'dashboardblocksview-ez-template'
+                    path: %ez_platformui.public_dir%/js/views/ez-dashboardblocksview.js
+                dashboardblocksview-ez-template:
+                    type: 'template'
+                    path: %ez_platformui.public_dir%/templates/dashboard-blocks.hbt
+                ez-dashboardblockbaseview:
+                    requires: ['ez-templatebasedview', 'dashboardblockbaseview-ez-template']
```

Comment:
again the base block is not supposed to be rendered, so it does not need a template. Please remove the template from the PR.